executor/adapter.go

```diff
@@ -1779,7 +1779,7 @@ func getEncodedPlan(stmtCtx *stmtctx.StatementContext, genHint bool) (encodedPla
 			// some hints like 'memory_quota' cannot be extracted from the PhysicalPlan directly,
 			// so we have to iterate all hints from the customer and keep some other necessary hints.
 			switch tableHint.HintName.L {
-			case "memory_quota", "use_toja", "no_index_merge", "max_execution_time",
+			case "memory_quota", "use_toja", "no_index_merge", "max_execution_time", "tidb_kv_read_timeout",
```

Comment:
Could we consider define a constant in `plannercore` scope, like the following several hints?